vuepress/docs/next/docs/reference/entandoapp-cr.md

```diff
@@ -25,13 +25,17 @@ spec:
 ## Specifications 
 | Spec Name | Description |
 | :- | :- |
-|  `ENTANDO_ECR_DEAPP_REQUEST_RETRIES`|  The number of times the Componenent Manager retries the component create/update process before quitting. Defaults to 3. |
-| `ENTANDO_ECR_DEAPP_REQUEST_BACKOFF` | The number of seconds to wait before the next 'create' attempt is executed. Defaults to 5. |
+|  `ENTANDO_ECR_DEAPP_REQUEST_RETRIES`| The number of times the Componenent Manager retries the component create/update process before quitting. Defaults to 3. |
+| `ENTANDO_ECR_DEAPP_REQUEST_BACKOFF` |The number of seconds to wait before the next 'create' attempt is executed. Defaults to 5. |
 | `ENTANDO_ECR_POSTINIT` | The configuration of the postinit process. |
-| `ENTANDO_CONTAINER_REGISTRY_CREDENTIALS` | The configuration for authenticated [OCI registries](../../tutorials/curate/bundle-private-images.md). |
-| `spec.customServerImage`|  Used to deploy the Docker image containing your custom Entando App. Follow these instructions on how to [build your own image](../../tutorials/devops/build-core-image.md). This property and the `spec.standardServerImage` are mutually exclusive.|
+| `ENTANDO_CONTAINER_REGISTRY_CREDENTIALS` |The configuration for authenticated [OCI registries](../../tutorials/curate/bundle-private-images.md). |
+| `spec.customServerImage`| The Docker image containing your custom Entando App. Follow these instructions to [build your own image](../../tutorials/devops/build-core-image.md). This property and the `spec.standardServerImage` are mutually exclusive.|
 |`spec.dbms` | Allowed values are: MySQL, PostgreSQL (default), Oracle, or embedded. Oracle is only supported as an external database.|
 |`spec.ecrGitSshSecretName`| The configuration used by the Entando Component Manager to download bundles from authenticated Git repositories. It's a Secret containing a private key file named `rsa_id` that matches a public key configured in the authenticated Git repository.|
+|`spec.environmentVariables`| A map of environment variables to pass to the EntandoApp Docker image. These variables pass to all components, and can sometimes be used as a mechanism to override any of the default environment variables that need customization.|
```

Comment:
@jyunmitch 

There is no `EntandoApp` docker image.

An EntandoApp is a only custom resource that is used to configure and manage 3 sub-modules (ComponentManager, AppEngine, AppBuilder) and their respective docker images.

The variables set under this property are propagated to all the 3 modules.

Reply:
that confused me a bit as well but it was there previously so I didn't mess with it. Will correct it now.